docs/translations.rst

```diff
@@ -19,10 +40,10 @@ The most common case is just wrapping text with ugettext.
         messages.success(request, _("Welcome!"))
 
 
-Sometimes it isn't always so simple, for example with report or settings
-section names. Two methods (`ugettext_noop` and `ugettext_lazy`) are available to mark
-a string for translation but not store the translated string value. These solve
-slightly different cases but are often interchangeable.
+When a string will be read by python as soon as the code is parsed, there is
```

Comment:
Would it be clear to say "Typically when code is run as a result of a module being imported, ..."?